bpython/curtsiesfrontend/repl.py

```diff
@@ -1577,9 +1580,9 @@ def move_screen_up(current_line_start_row):
         else:
             cursor_row, cursor_column = divmod(
                 (
-                    len(self.current_cursor_line_without_suggestion)
-                    - len(self.current_line)
-                    + self.cursor_offset
+                    self.current_cursor_line_without_suggestion.width
+                    - wcswidth(self.current_line)
```

Comment:
What's the distinction here between the wcswidth and .width, are some of these FmtStrs and some of them plain strs? (and if it's not clear, could you add comments to the definitions or initialization lines?)